library/standard_tables_capture.inc

```diff
@@ -243,6 +244,144 @@ function snomed_import($us_extension=FALSE) {
     return true;
 }
 
+function snomedRF2_import($us_extension=FALSE) {
+
+    // set up array
+    $table_array_for_snomed=array(
+        "sct2_concept_drop"=>"DROP TABLE IF EXISTS `sct2_concept`",
+        "sct2_concept_structure"=>"CREATE TABLE IF NOT EXISTS `sct2_concept` (
+            `id` bigint(20) NOT NULL,
+            `effectiveTime` date NOT NULL,
+            `active` int(11) NOT NULL,
+            `moduleId` bigint(20) NOT NULL,
+            `definitionStatusId` bigint(25) NOT NULL,
+             PRIMARY KEY (`id`)
+            ) ENGINE=InnoDB",
+        "sct2_description_drop"=>"DROP TABLE IF EXISTS `sct2_description`",
+        "sct2_description_structure"=>"CREATE TABLE IF NOT EXISTS `sct2_description` (
+            `id` bigint(20) NOT NULL,
+            `effectiveTime` date NOT NULL,
+            `active` bigint(11) NOT NULL,
+            `moduleId` bigint(25) NOT NULL,
+            `conceptId` bigint(20) NOT NULL,
+            `languageCode` varchar(8) NOT NULL,
+            `typeId` bigint(25) NOT NULL,
+            `term` varchar(255) NOT NULL,
+            `caseSignificanceId` bigint(25) NOT NULL,
+             PRIMARY KEY (`id`)
+            ) ENGINE=InnoDB",
+        "sct2_identifier_drop"=>"DROP TABLE IF EXISTS `sct2_identifier`",
+        "sct2_identifier_structure"=>"CREATE TABLE IF NOT EXISTS `sct2_identifier` (
+            `identifierSchemeId` bigint(25) NOT NULL,
+            `alternateIdentifier` bigint(25) NOT NULL,
+            `effectiveTime` date NOT NULL,
+            `active` int(11) NOT NULL,
+            `moduleId` bigint(25) NOT NULL,
+            `referencedComponentId` bigint(25) NOT NULL,
+             PRIMARY KEY (`identifierSchemeId`)
+            ) ENGINE=InnoDB",
```

Comment:
Note sure why, but this table isn't getting populated with any data on the import.